LEGO1/lego/legoomni/src/video/legovideomanager.cpp

```diff
@@ -125,14 +125,18 @@ MxResult LegoVideoManager::Create(MxVideoParam& p_videoParam, MxU32 p_frequencyM
 
 	m_direct3d->SetDevice(deviceEnumerate, driver, device);
 
-	if (!driver->m_ddCaps.dwCaps2 && driver->m_ddCaps.dwSVBRops[7] != 2) {
-		p_videoParam.Flags().SetF2bit0(TRUE);
+	/*
+	 * BUG: should be:
+	 *  if ((driver->m_ddCaps.dwCaps2 & DDCAPS2_CERTIFIED) == 0 && driver->m_ddCaps.dwSVBRops[7] != 2) {
+	 */
+	if (driver->m_ddCaps.dwCaps2 != DDCAPS2_CERTIFIED && driver->m_ddCaps.dwSVBRops[7] != 2) {
+		p_videoParam.Flags().SetBrokenLighting(TRUE);
```

Comment:
I don't think we should embed the fact that the condition is broken into the naming of the function :D  I'd suggest `SetDisableLighting`, as that seems to be what it's doing

Reply:
Ha ha, no that name isn't related to the borken code, we could call it GetLacksLightSupport() instead to make that more clear.

Reply:
Oh I see. Yes, let's call it something other than "broken". `LacksLightSupport` sounds reasonable

Reply:
Done